### PR TITLE
Issue_termplate

### PR DESCRIPTION
**Pull request by wsdjeg**
It is new feature of github,when open a new issue ,the issue template will be auto type in.should we use this feature?
https://github.com/blog/2111-issue-and-pull-request-templates

### CODE REVIEW

**Comment by artur-shaik** [comment]
Yeah, of cause. I just looked for some free time to create it. So, thanks. Will look what we have there...

**Comment by artur-shaik** [comment]
I think here we need help of @brcolow and @adelarsq

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+## Provide a minimal .vimrc with less than 50 lines (Required!)
+
+```vim
+" Your minimal .vimrc
+set runtimepath^=~/path/to/vim-javacomplete2/
+autocmd FileType java setlocal omnifunc=javacomplete#Complete
+```
+
+
+## The reproduce ways from Vim starting (Required!)
```
Comment (brcolow) [inline .github/ISSUE_TEMPLATE.md]: The steps taken to reproduce the bug (Required!)

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+ 1. foo
+ 2. bar
+ 3. baz
+
+
+## Screen shot (if possible)
+
+
+## Upload the log messages by `:redir` and `:message`
```
Comment (brcolow) [inline .github/ISSUE_TEMPLATE.md]: The output of `:redir` and `:message`

**Comment by brcolow** [comment]
Left a couple comments, otherwise looks pretty good. My notes are pretty subjective and a native English speaker would still understand the meaning without these changes, I just tried to make it sound a bit more natural.

**Comment by adelarsq** [comment]
Nice feature. I will add here to discuss and preview before send a pull request. I did some changes on the @brcolow template. My english is pretty bad, so I can help with that :yum: 

For the issue template:

----

#### :open_mouth: Actual behavior:
...
#### :blush: Expected behavior:
...
#### :footprints:  Steps to reproduce the behavior:
...
#### :shell: Log messages and Screenshots if avaiable:
...
#### :computer: Environment where the behavior occurs:
Mark, put version and Operation System for each editor used:
[ ] Vim
[ ] Gvim
[ ] Macvim
[ ] Neovim
[ ] Other?

---

For the pull request template:

---

#### :rocket: Changes proposed:
...

#### :white_check_mark: Fix for the issues:
#xxx, #xxx, etc

---

**Comment by wsdjeg** [comment]
https://github.com/Valloric/YouCompleteMe/blob/master/.github/ISSUE_TEMPLATE.md

**Comment by adelarsq** [comment]
https://github.com/devspace/awesome-github-templates

**Comment by wsdjeg** [comment]
@adelarsq maybe no need add emoj,because when a people open a new issue,he can not see the icons,only something just like `:smile:`

**Comment by adelarsq** [comment]
@wsdjeg I did create another version without. The objective is add a visual context for each section, so visual people feel better. But yes, have to use the preview.

I will see the [youcompleteme](https://github.com/Valloric/YouCompleteMe/blob/master/.github/ISSUE_TEMPLATE.md) and [awesome-github-templates](https://github.com/devspace/awesome-github-templates) template to improve.

**Comment by artur-shaik** [comment]
Ok, guys. I decided not to use emoji. 

So, feel free to comment or PR you suggestions.

**Pull request merged**